keylime/mba/elparsing/tpm2_tools_elparser.py

```
@@ -83,13 +83,20 @@ def parse_binary_bootlog(log_bin: bytes) -> typing.Tuple[Failure, typing.Optiona
-    if len(retDict_tpm2["reterr"]) > 0:
+    if retDict_tpm2["code"] != 0:
         failure.add_event(
-            "tpm2_eventlog.warning",
-            {"context": "tpm2_eventlog exited with warnings", "data": str(retDict_tpm2["reterr"])},
+            "tpm2_eventlog.error",
+            {
+                "context": "tpm2_eventlog exited with non-zero return code",
+                "code": retDict_tpm2["code"],
+                "data": str(retDict_tpm2["reterr"]),
+            },
             True,
```

Ruff (0.15.9)

[warning] Boolean positional value in function call

(FBT003)